api/batch/resource_test.go

```diff
@@ -26,12 +26,16 @@ import (
 	"github.com/gojek/merlin/models"
 )
 
+const (
+	environmentName  = "dev"
+	orchestratorName = "merlin"
+)
```

Comment:
Sprinkled this here and there wherever I happened to see repetition of the environment name or orchestrator name in the tests.